lib/pure/smtp.nim

```diff
@@ -167,6 +181,17 @@ proc connect*(smtp: Smtp | AsyncSmtp,
   await smtp.checkReply("220")
   await smtp.debugSend("HELO " & address & "\c\L")
   await smtp.checkReply("250")
+  
+proc starttls*(smtp: Smtp | AsyncSmtp, sslContext: SSLContext = nil) {.multisync.} =
```

Comment:
This procedure is exported and so needs to be documented, please add docs similar to how the other procedures are documented.

Reply:
:+1: